requirements/generated/requirements-production.txt

```diff
@@ -91,13 +91,13 @@ https://github.com/ministryofjustice/djorm-ext-pgfulltext/archive/refs/tags/0.1.
     # via -r requirements/source/requirements-base.in
 docopt==0.6.2
     # via django-docopt-command
-drf-extensions==0.2.5
+drf-extensions==0.2.8
     # via -r requirements/source/requirements-base.in
 drf-nested-routers==0.9.0
     # via -r requirements/source/requirements-base.in
 et-xmlfile==1.0.1
     # via openpyxl
-futures==3.3.0
+futures==3.4.0
```

Comment:
Why did this change?

Reply:
Whatever is depending on this has not pinned a minor version and there has been a new 3.4 release of the futures package in October 2022 https://pypi.org/project/futures/3.4.0/